editor/import_dock.cpp

```diff
@@ -423,6 +423,7 @@ ImportDock::ImportDock() {
 	import_opts = memnew(EditorInspector);
 	add_child(import_opts);
 	import_opts->set_v_size_flags(SIZE_EXPAND_FILL);
+	import_opts->set_enable_capitalize_paths(bool(EDITOR_GET("interface/inspector/capitalize_properties")));
```

Comment:
CC @akien-mga Maybe we should rename this path "interface/inspector/capitalize_properties" to a more generic.

Reply:
Or should I use a different names for this?

Reply:
I think it should default to true in EditorInspector.

Reply:
Changed.